.github/workflows/ci.yml

```diff
@@ -42,6 +42,21 @@ jobs:
           project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
           coverage-reports: ./target/scala-3.3.0/coverage-report/cobertura.xml
 
+  test-docs-build:
+    name: Test docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+      - name: Install requirements (pip, npm, apt)
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r docs/requirements.txt
+          npm install --global typedoc
+          sudo apt-get install graphviz
```

Comment:
why is graphviz added here? if that's copy-pasted from the API repo, then it shouldn't be needed here, I think. In the API repo we have `.dot` files for some diagrams, that's why we need it there. But generally, I'd opt for mermaid diagrams instead of `.dot` anyway, so even if we need diagrams, I don't think we need graphviz.

Reply:
I have copied over the setup from `dsp-api` and only added the `neoteroi-mkdocs`. I was not aiming to clean up the setup and do not know which dependencies is used where.

+1 for me to only use `mermaid` for diagrams.

I have remove both `graphvic` and `typdoc` from the `ci.yml` and the documentation still builds: https://github.com/dasch-swiss/dsp-ingest/pull/26/commits/68b19cd9badcbef4f6231a029908a5d36e011dcd